app/views/posts/show.html.erb

```diff
@@ -3,25 +3,33 @@
     <div class="post">
         <ul class="post_info width"> 
             <div class="header">
-                <h3 class="title"><%= @posts.title %> #<%= @posts.id%> by <%= @user.name %></h3>
-                <li class="countcomentlikes">Comments:  <%= @posts.comments_counter %>, Likes: <%= @posts.likes_counter %> </li>
+                <h3 class="title"><%= @post.title %> #<%= @post.id%> by <%= @user.name %></h3>
+                <li class="countcomentlikes">Comments:  <%= @post.comments_counter %>, Likes: <%= @post.likes_counter %> </li>
             </div>
-            <li class="bio_content"> <%= @posts.text %> </li>
+            <li class="bio_content"> <%= @post.text %> </li>
         </ul>
         <button class="page-btn">
-            <%= link_to 'Add New Comment', new_user_post_comment_path(@posts.author, @posts), style: 'color: white; text-decoration: none;' %>
+            <%= link_to 'Add New Comment', new_user_post_comment_path(@post.author, @post), style: 'color: white; text-decoration: none;' %>
         </button>
```

Comment:
I am unable to create a new comment on any post. Every time I click the `Add new comment` button nothing happens.

- Kindly look into the authorization rules or perhaps the comments create action and make sure that every user is authorized to comment on any post including posts they do not own.

Reply:
<img width="1440" alt="Screenshot 2023-06-23 at 15 59 31" src="https://github.com/Tafloninno/Blog-App/assets/111200646/c7e805c7-252f-4bed-addd-bbd731d7e1b5">